lms/djangoapps/program_enrollments/api/v1/tests/test_views.py

```diff
@@ -171,3 +173,191 @@ def test_200_many_pages(self):
             assert next_response.data['previous'] is not None
             assert self.get_url(self.program_uuid) in next_response.data['previous']
             assert '?cursor=' in next_response.data['previous']
+
+
+class ProgramEnrollmentViewPostTests(APITestCase):
+    """
+    Tests for the ProgramEnrollment view POST method.
+    """
+
+    def test_successful_program_enrollments(self):
+        global_staff = GlobalStaffFactory.create(username='global-staff', password='password')
```

Comment:
The creation of global staff and login statements can be moved into `setUp` method.
For the `test_unauthenticated` case, you can just invoke `self.client.logout()`

Reply:
Sweet! Thanks for the `logout` suggestion